modules/configuration/src/main/java/org/apache/ignite/configuration/storage/Data.java

```diff
@@ -56,14 +51,7 @@ public Data(Map<String, Serializable> values, long cfgVersion, long storageRevis
      * Get version.
      * @return version.
      */
-    public long cfgVersion() {
-        return cfgVersion;
-    }
-
-    /**
-     * @return Storage revision.
-     */
-    public long storageRevision() {
-        return storageRevision;
+    public long changeId() {
```

Comment:
This name was proposed by @ibessonov, it is better to discuss with him naming. About version, this `changeId` also valid for `LocalConfigurationSrorage` as far as the mechanism for updating configuration is common. For LocalConfigurationSrorage` it is just a counter for configuration updates, however for `DistributedConfigurationSrorage` it is a metastorage revision, which is actual for the current update of configuration. I don't think that it is reasonable to refer to that in `changeId` javadoc because this is specific for the different configuration storages

Reply:
Ok, it is up to you, but I don't think that the Javadoc in its current state provides any valuable information about this `changeId`.